Dockerfile

```diff
@@ -144,4 +144,13 @@ RUN sudo apt-get -y install gcovr && \
 # Add engineering tools scripts to image
 ADD --chown=carma ./code_coverage /home/carma/.ci-image/engineering_tools/code_coverage
 
-ENTRYPOINT [ "/home/carma/.base-image/entrypoint.sh" ]
+RUN sudo apt-get install -y sqlite3 autotools-dev automake
```

Comment:
I'd add these to the list on line 23 rather than in their own line, that adds an extra layer to the final image that we don't really need.